clients/fides-js/src/services/api.ts

```diff
@@ -22,12 +25,31 @@ export const fetchExperience = async (
   userLocationString: string,
   fidesApiUrl: string,
   debug: boolean,
+  apiOptions?: FidesApiOptions | null,
```

Comment:
we can do this later, but I think we either need to parametrize `fetchExperience`'s arguments, or pass in the whole `options` object. there are too many positional arguments!